html/includes/functions.inc.php

```diff
@@ -12,6 +12,29 @@
  * @copyright  (C) 2013 LibreNMS Group
  */
 
+function var_isset($v) {
```

Comment:
I would definitely prefer to see var_isset gone, since it makes the code slightly less obvious, only saves one character, and adds the overhead of a function call.